.github/workflows/ci.yml

```diff
@@ -21,6 +21,12 @@ jobs:
    
     - name: Checkout
       uses: actions/checkout@v2
+
+    - uses: swift-actions/setup-swift@v1
+      with:
+        swift-version: "5.8.0"
+    - name: Get swift version
+      run: swift --version
```

Comment:
why do we need the swift action? 
for running tests we can also use the `xcodebuild` command:
See an example here: https://mokacoding.com/blog/running-tests-from-the-terminal/